Tests/MessagingInApp/Gist/Managers/MessageQueueManagerTest.swift

```diff
@@ -64,6 +64,45 @@ class MessageQueueManagerTest: UnitTest {
         XCTAssertEqual(actualMessages, [givenMessage2, givenMessage1, givenMessage3])
     }
 
+    func test_getInlineMessages_givenQueueMessageHasPageRule_pageRuleMatch_expectInlineMessage() {
+        let givenElementId1 = String.random
+        let givenElementId2 = String.random
+        let givenElementId3 = String.random
+        let currentRoute = String.random
+
+        let givenMessage1 = Message(pageRule: currentRoute, elementId: givenElementId1, priority: 1)
+        let givenMessage2 = Message(pageRule: currentRoute, elementId: givenElementId2, priority: 0)
+        let givenMessage3 = Message(elementId: givenElementId3, priority: 0)
+
+        manager.localMessageStore = [
+            "1": givenMessage1,
+            "2": givenMessage2,
+            "3": givenMessage3
+        ]
+        Gist.shared.setCurrentRoute(currentRoute)
+        let actualMessages = manager.getInlineMessages(forElementId: givenElementId1)
+        XCTAssertEqual(actualMessages, [givenMessage1])
+    }
+
+    func test_getInlineMessages_givenQueueMessageHasPageRule_pageRuleMisMatch_expectEmptyArray() {
```

Comment:
```suggestion
    func test_getInlineMessages_givenQueueMessageHasPageRule_givenNoSetCurrentRoute_expectEmptyArray() {
```

I think this is a good test function, testing if `Gist.shared.setCurrentRoute(currentRoute)` is not called. 

The test function name made me think that we were testing if a page rule is set, but the Messages have a page rule *not* matching the currently set route in the SDK.